interface/patient_file/summary/track_anything_fragment.php

```diff
@@ -42,7 +42,7 @@
 } else {  // We have some tracks here...
     echo "<span class='text'>";
     echo xlt('Available Tracks') . ":";
-    echo text($result);
+    echo json_encode($result);
```

Comment:
is there a better way to escape an array here @bradymiller ?

![Screenshot from 2022-10-26 14-28-11](https://user-images.githubusercontent.com/5618372/198107377-7a2621ce-eba9-42fb-b5aa-4359186a3a0d.png)

Reply:
oh, nvm, this is trivial

Reply:
That is an odd line where was basically echoing an array.
is that line even needed, since basically the same query is done below that and then outputted in a human readable form?

Reply:
i think can remove lines 45 and 47